src/factories/series/Column.ts

```diff
@@ -0,0 +1,92 @@
+module n3Charts.Factory.Series {
```

Comment:
I agree on that. you are referring to the `module n3Charts.Factory.Series` so I guess you are speaking of the `SeriesOptions` class?